packages/ricos-viewer/web/src/modals/fullscreen/FullscreenProvider.tsx

```diff
@@ -34,8 +35,27 @@ export default class FullscreenProvider extends Component<Props, State> {
     };
   }
 
+  getImagesCount(): number {
+    if (this.props.initialState) {
+      return getImagesData(this.props.initialState).images.length;
+    } else {
+      return 0;
+    }
+  }
+
   componentDidMount() {
-    this.loadEditorModalAfterLocaleResourceIsLoadedToPreventRemountHackFromBreakingModal();
+    if (this.props.initialState && this.getImagesCount() > 0) {
```

Comment:
remove this.props.initialState
```suggestion
    if (this.getImagesCount() > 0) {
```